tests/common/tensor/test_tensor.py

```diff
@@ -26,39 +26,6 @@ def tearDown(self):
         """Hook method for deconstructing the test fixture after testing it."""
         pass
 
-    @decorator.cuda_test
```

Comment:
Removing this UT means we support to cast to kbfloat16?

Reply:
Yes. The PR supports all conversion (refer to the function `test_tensor_cast`).